tools/automate_bindings.py

```diff
@@ -70,11 +82,29 @@ def getSelf(module_name):
     '''
     return frame % (module_name, module_name)
 
+def findCorrespondingKeyInDict(dictionary, full_key):
+    for key in dictionary:
+        if full_key.find(key) is not -1:
```

Comment:
1. Numbers should be compared with `==`, not with `is`.
2. `if key in full_key:`